src/pages/Admin/Charity/Permissions/Form.tsx

```diff
@@ -0,0 +1,43 @@
+import { ErrorMessage } from "@hookform/error-message";
+import { getTypedKeys } from "helpers";
+import MobileTable from "./MobileTable";
+import Table from "./Table";
+import useSubmit from "./useSubmit";
+
+export default function Form() {
+  const { isSubmitting, errors, reset, submit } = useSubmit();
+
+  const errorName = getTypedKeys(errors).find((errKey) => !!errors[errKey]);
+
+  return (
+    <form className="grid gap-6 w-full" onSubmit={submit} onReset={reset}>
+      <Table className="max-lg:hidden" />
+      <MobileTable className="lg:hidden" />
+
+      <div className="flex flex-wrap items-center gap-3">
+        <button
+          type="reset"
+          disabled={isSubmitting}
+          className="btn-outline-filled py-2 px-8 rounded"
+        >
+          Reset
```

Comment:
@misicnenad, `reset` isn't clear vs its wiring
say I have this starting 
<img width="1164" alt="image" src="https://user-images.githubusercontent.com/89639563/226302491-44bf6cfa-b308-4951-9bd2-9f8b093b0e98.png">

when I click reset: 
it removes the delegated addresses but `delegate` is still checked
<img width="1163" alt="image" src="https://user-images.githubusercontent.com/89639563/226302583-2a7b4f06-bf49-43ab-9e71-838c6aa50b03.png">

if `reset` pertains to resetting changes, then after clicking 
should just revert to original data when first going to permissions form

on the other hand if `reset` is pertains to `default`(unconfigured)
then should reset to this  state
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/89639563/226303126-c7f99613-88a4-44ef-90b8-000b18d09472.png">

we can't go halfway between since it's confusing

Reply:
@misicnenad `Reset` should revert the form to the last known on-chain settings that we get from the contract when the page loads.

Reply:
@misicnenad, REPRO:
* go to `admin/27/permissions`
* click `reset`

Reply:
> @misicnenad `Reset` should revert the form to the last known on-chain settings that we get from the contract when the page loads.

Was expecting it to do as much, looking into this.

Reply:
@SovereignAndrey @ap-justin the issue was using `<button type="reset" ...` and `<form onReset={...`, seems that `react-hook-form` doesn't know how to handle resetting fields when implemented this way. 
Updated to use `<button type="button" onClick={reset}...` logic (as is used usually) https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1885/commits/a3a2c96759e889b1d5ad5b74066a3b54d8de9741